modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Bare Metal Operator]
+ifdef::cluster-caps[= Bare-metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Bare Metal Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
```

Comment:
"Cluster Baremetal Operator" and "Baremetal operator" are two different thing

Reply:
@honza Thank you for clarifying. I see that I conflated the BMO and CBO, probably because there is not much information about the CBO in the OpenShift docs.

Does the Cluster Baremetal Operator provide the functionality for the `baremetal` capability? Is this repo the best source to flesh out the description for the baremetal capability: https://github.com/openshift/cluster-baremetal-operator?

Reply:
I looked over the information at https://github.com/openshift/cluster-baremetal-operator, and I think I addressed your concerns. Would you please take another look and let me know if more changes are required?

Reply:
@honza Would you please take a look at the updates?